Shear/HistoryType.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  HistoryType.swift
+//  Shear
+//
+//  Created by Andreas Kompanez on 23.02.20.
+//  Copyright © 2020 Andreas Kompanez. All rights reserved.
+//
+
+import Foundation
+
+protocol HistoryType {
+    var storage: HistoryStorageType { get }
+}
+
+
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/kmpnz/Shear/blob/895e50bbb85826ea77037e4a1c1e5468d960c4fc/Shear/HistoryType.swift#L15"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>HistoryType.swift:15</code></del></td>
    </tr>
  </tbody>
</table>